Makefile

```diff
@@ -20,6 +20,9 @@ format:
 	go vet ./pkg/...
 
 test:
+	hack/tests.sh
+
+docker-test:
```

Comment:
it should be the other way around - docker-test runs hack/tests.sh

Reply:
Done